layouts/shortcodes/events.html

```diff
@@ -0,0 +1,4 @@
+<div class="col-sm-8 col-sm-offset-1 news-list">
```

Comment:
Are you using this new shortcode anywhere? If not, we should remove them.

Reply:
Removed the shortcode